### PR TITLE
Fix various linting issues

### PR DESCRIPTION
**Pull request by arbron**
Just a few code style cleanup issues.

### CODE REVIEW

File: module/applications/actor/base-sheet.mjs [hunk 1]
```diff
@@ -402,7 +402,7 @@ export default class ActorSheet5e extends ActorSheet {
       const physical = [];
       if ( data.bypasses?.size ) {
         values = values.filter(t => {
-          if ( !CONFIG.DND5E.physicalDamageTypes[t] ) return true;
+          if ( !globalThis.CONFIG.DND5E.physicalDamageTypes[t] ) return true;
```
Comment (Fyorl) [inline module/applications/actor/base-sheet.mjs:405]: Not sure the `globalThis` prefix is necessary to access `CONFIG` here, or below.

**Review by Fyorl — CHANGES_REQUESTED** [review]

File: module/applications/actor/base-sheet.mjs (hunk 1 above)
Comment (arbron) [reply module/applications/actor/base-sheet.mjs:405]: It’s the only way I could find that would fix the `no-loop-func` ESLint warning that was being incorrectly thrown here. The other option would be to manually annotate it to skip that warning.

Comment (Fyorl) [reply module/applications/actor/base-sheet.mjs:405]: Honestly, I might vote to just turn off that warning entirely. I don't think I've ever had it flag something that wasn't a false positive. What do you think?

Comment (arbron) [reply module/applications/actor/base-sheet.mjs:405]: I have no objections to that :)

**Review by Fyorl — APPROVED** [review]

**Pull request merged**